lib/rules/indent.js

```diff
@@ -1075,7 +1075,9 @@ module.exports = {
                  * var foo = bar &&
                  *                   baz;
                  */
-
+                if (operator === null) {
```

Comment:
This comment is a bit confusing now because it looks like it's referring to this line.